### DISCUSSION TITLE
Scores missing within Lazer

### DISCUSSION THREAD

**Discussion by xVenora** [ppy/osu#25397] [Q&A] [answered]
So i decided to test out lazer today, and i noticed my top play was missing from my profile.
After a little digging i discovered i'm not the only one it happened to.
Others are also missing some of their top plays. Everything was set on stable. 

### **Some screenshots to clarify**

<details>
<summary>My Stable profile:</summary>
<img src="https://github.com/ppy/osu/assets/44437752/af838cb8-26da-428c-9eeb-b79c95cedabe">
</details>

<details>
<summary>My Lazer profile:</summary>
<img src="https://github.com/ppy/osu/assets/44437752/aa1bbd29-0df9-45a1-8de9-a91471391dae">
</details>

<details>
<summary>Some random dudes profile i found who's now rank 1 on lazer:</summary>
<img src="https://github.com/ppy/osu/assets/44437752/27baa6fc-5f05-4ed1-aa15-bee69d645453">
<img src="https://github.com/ppy/osu/assets/44437752/ec688c02-f87c-4f03-8acb-76ea2c09b3c4">
</details>

<details>
<summary>The same dudes profile on stable:</summary>
<img src="https://github.com/ppy/osu/assets/44437752/72a9ed83-0822-42b1-a09a-f0784eb3898a">
<img src="https://github.com/ppy/osu/assets/44437752/3fcd808e-9747-4ab0-af24-ec061c1794b5">
</details>

<details>
<summary>And just to be sure here's proof that i didn't make this up:</summary>
<img src="https://github.com/ppy/osu/assets/44437752/04cb1b43-3271-41bc-9eae-ee186cf80f90">
<img src="https://github.com/ppy/osu/assets/44437752/a968f860-a8c6-4c21-8fbe-9c602cac87a2">
</details>

Now i don't know if my client is just having a stroke but it's probably worth looking into either way.
Hope you can work with this :)

**Comment by peppy** [comment]
Looks to be new scores only. As far as I can see, the processes required are running.

@xVenora  To start with, could you please link your profile, rather than providing non-useful screenshots of it?

cc/ @ppy/team-web

**Reply by xVenora** [reply]
oh yeah of course
https://osu.ppy.sh/users/30149051

**Reply by peppy** [reply]
Thanks.

```sql

root@db-master:osu> select * from osu_scores_high where score_id = 4515277954\G;
***************************[ 1. row ]***************************
score_id        | 4515277954
beatmap_id      | 809469
user_id         | 30149051
score           | 93599830
maxcombo        | 2035
rank            | S
count50         | 0
count100        | 13
count300        | 1488
countmiss       | 0
countgeki       | 234
countkatu       | 12
perfect         | 0
enabled_mods    | 16
date            | 2023-10-02 19:21:02
pp              | 501.959
replay          | 1
hidden          | 0
country_acronym | DE


root@db-master:osu> select * from scores where id = 2434470320\G;
***************************[ 1. row ]***************************
id         | 2434470320
user_id    | 30149051
beatmap_id | 809469
ruleset_id | 0
data       | {"mods": [{"acronym": "HR"}, {"acronym": "CL"}], "rank": "S", "passed": true, "user_id": 30149051, "accuracy": 0.9942260715078836, "ended_at": "2023-10-02T19:21:02+00:00", "max_combo": 2035, "beatmap_id": 809469, "statistics": {"ok": 13, "great": 1488}, "total_score": 997924, "legacy_score_id": 4515277954, "legacy_total_score": 93599830, "maximum_statistics": {"great": 1501, "legacy_combo_increase": 535}}
has_replay | 0
preserve   | 1
created_at | 2023-10-02 19:21:02
updated_at | 2023-10-02 19:21:02


root@db-master:osu> select * from score_performance where score_id = 2434470320\G;
***************************[ 1. row ]***************************
score_id | 2434470320
pp       | 0.0

```

To my eye, it looks like the score was imported to the new table structure before PP calculation had completed. Likely something we haven't accounted for. I'll investigate further.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu-queue-score-statistics/pull/167
